Doc/library/sys.rst

```diff
@@ -1368,13 +1368,26 @@ always available.
    returned by the :func:`open` function.  Their parameters are chosen as
    follows:
 
-   * The character encoding is platform-dependent.  Under Windows, if the stream
-     is interactive (that is, if its :meth:`isatty` method returns ``True``), the
-     console codepage is used, otherwise the ANSI code page.  Under other
-     platforms, the locale encoding is used (see :meth:`locale.getpreferredencoding`).
-
-     Under all platforms though, you can override this value by setting the
-     :envvar:`PYTHONIOENCODING` environment variable before starting Python.
+   * The character encoding is platform-dependent.  Non-Windows
+     platforms use the locale encoding (see
+     :meth:`locale.getpreferredencoding()`).
+     On Windows, UTF-8 is used for console character
+     devices (i.e. CON, CONIN$, and CONOUT$).  However, this
+     can be overridden to use the console as a generic
+     character device by setting the environment variable
+     :envvar:`PYTHONLEGACYWINDOWSSTDIO` before starting Python.  Non-
+     character devices such as disk files and pipes use the
+     system locale encoding (i.e. the ANSI codepage).
+     Character devices such as NUL (i.e. isatty() returns
+     True) use the value of the console input and output
+     codepages at startup, respectively for stdin and
+     stdout/stderr.  This defaults to the system locale
+     encoding if the process is not initially attached to a
+     console.
+     Under all platforms, you can override this value by
```

Comment:
Might need clarifying that “this value” is the character encoding. Put the blank line back to make a separate paragraph, or try “you can override the character encoding by . . .”

Reply:
Changed it.